dart/lib/src/transport/http_transport.dart

```diff
@@ -39,20 +45,24 @@ class HttpTransport implements Transport {
   }
 
   @override
-  Future<SentryId> send(SentryEvent event) async {
-    final data = event.toJson();
+  Future<SentryId> sendSentryEvent(SentryEvent event) async {
+    final envelope = SentryEnvelope.fromEvent(event, _options.sdk);
+    return await sendSentryEnvelope(envelope);
+  }
```

Comment:
iirc we've agreed that the Transport layer would only have:

`Future<SentryId> send(SentryEnvelope envelope)`

and SentryClient always wraps up to an envelope before calling the Transport layer

so `SentryEnvelope.fromEvent(event, _options.sdk);` should be done by SentryClient instead

Reply:
ops addressed by https://github.com/getsentry/sentry-dart/pull/426/files my bad